crates/fs/common/error.rs

```diff
@@ -56,8 +56,11 @@ pub enum FsError {
     #[error("Expected bare ratchet key")]
     ExpectBareRatchetKey,
 
-    #[error("Missing unix fs metadata field")]
-    MissingUnixFSMetadata,
+    #[error("Missing created time metadata")]
+    MissingCreatedTimeMetadata,
+
+    #[error("Missing modified time metadata")]
+    MissingModifiedTimeMetadata,
```

Comment:
No change for this PR needed - but we should probably make timestamps in the metadata optional.

Reply:
Right. It is an easy change to make. Let me fix it.